### ISSUE TITLE
Log data accumulation causing excessive use of computational resources (hard disk).

### ISSUE THREAD

**Issue by matheusmts101** [browserless/browserless#4011]
Good morning,

We use the browserless solution for simultaneous use of clients with one or more headless browsers. In our case, we use version V1 due to code limitations and the removal of some features in V2 that significantly impact usage, making migration not feasible at this time.

As we analyze usage over time, we have noticed that the accumulation of cache, logs, and metrics can be significant (we keep the browser open all the time, closing it can disrupt user usage and cause disconnection from the tool used within the browser).

Here is an overview of the files and directories within Browserless that are taking up disk space:

Browserless connection folder
chrome_debug.log file
BrowserMetrics-spare.pma file
default folder
GPUCache folder
Local Storage folder
blob_storage folder
'Code Cache' folder
chrome_debug.log file
Cache folder
IndexedDB folder
MANIFEST-11034093 file
file with .ldb extension
Service Worker folder
CacheStorage folder

Many of these files and directories can grow to several gigabytes in size (some reaching nearly 200 GB), which can negatively impact the user experience.

In an attempt to disable some of these features, we followed the documentation and tried the following commands:

$ docker run -e "ENABLE_DEBUGGER=false" -p 3000:3000 --restart always -d --name browserless browserless/chrome
$ docker run -e "DEBUG=*" -p 3000:3000 --restart always -d --name browserless browserless/chrome
We also set these configurations directly in the dockercompose file.

These changes appeared as active Envs, but upon testing, the listed folders and files continued to store cache.

We would appreciate any suggestions on how to disable this cache accumulation without the need for cron jobs, as this would impact resource usage (since we are using the solution in a cloud environment).

![image](https://github.com/browserless/browserless/assets/117188864/0bdb2220-004e-4d48-b927-d1b40c06251e), 
![image](https://github.com/browserless/browserless/assets/117188864/9fcde102-7d1a-485e-92c3-22f6102f773c)
![image](https://github.com/browserless/browserless/assets/117188864/e6ccb088-7950-4a38-b15a-d7078dabffa7)

**Comment by matheusmts101** [comment]
Good morning,

We use the browserless solution for simultaneous use of clients with one or more headless browsers. In our case, we use version V1 due to code limitations and the removal of some features in V2 that significantly impact usage, making migration not feasible at this time.

As we analyze usage over time, we have noticed that the accumulation of cache, logs, and metrics can be significant (we keep the browser open all the time, closing it can disrupt user usage and cause disconnection from the tool used within the browser).

Here is an overview of the files and directories within Browserless that are taking up disk space:

Browserless connection folder
chrome_debug.log file
BrowserMetrics-spare.pma file
default folder
GPUCache folder
Local Storage folder
blob_storage folder
'Code Cache' folder
chrome_debug.log file
Cache folder
IndexedDB folder
MANIFEST-11034093 file
file with .ldb extension
Service Worker folder
CacheStorage folder

Many of these files and directories can grow to several gigabytes in size (some reaching nearly 200 GB), which can negatively impact the user experience.

In an attempt to disable some of these features, we followed the documentation and tried the following commands:

$ docker run -e "ENABLE_DEBUGGER=false" -p 3000:3000 --restart always -d --name browserless browserless/chrome
$ docker run -e "DEBUG=*" -p 3000:3000 --restart always -d --name browserless browserless/chrome
We also set these configurations directly in the dockercompose file.

These changes appeared as active Envs, but upon testing, the listed folders and files continued to store cache.

We would appreciate any suggestions on how to disable this cache accumulation without the need for cron jobs, as this would impact resource usage (since we are using the solution in a cloud environment).
